server/tests/integrations/stripe/test_payment_dunning.py

```diff
@@ -0,0 +1,106 @@
+import uuid
```

Comment:
For tests, what I usually do is to mirror the name of the module. For example:

* `polar/order/service.py` -> `tests/order/test_service.py`
* `polar/integrations/stripe/payment.py` -> `tests/integrations/stripe/test_payment.py`

Reply:
Done